tests/Unit/Elliptic/Utilities/Test_ApplyAt.cpp

```diff
@@ -0,0 +1,108 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <map>
+#include <string>
+#include <tuple>
+#include <unordered_map>
```

Comment:
Do you want test this with a `DirectionMap` as well? I don't see why it wouldn't work, but might not be bad to test. Up to you!

Reply:
added